src/Core/Utility.js

```diff
@@ -146,6 +162,19 @@ function assign (object, a, b) {
 	return object
 }
 
+/**
+ * @param {object} object
+ * @param {object} a
+ * @return {object}
+ */
+function pickout (object, a) {
```

Comment:
rename to `defaults` to match [lodash](https://lodash.com/docs/4.17.10#defaults)